movies.md

```diff
@@ -260,6 +261,14 @@
 * Country: UK, USA
 * [_IMDb_](http://www.imdb.com/title/tt0113189/)
 
+#### Steve Jobs
+>Steve Jobs takes us behind the scenes of the digital revolution, to paint a portrait of the man at its epicenter. The story unfolds backstage at three iconic product launches, ending in 1998 with the unveiling of the iMac.
+
+![Steve_Jobs](assets/stevejobs.jpg)
```

Comment:
Thanks for PR, just one thing before merged it I'd like to use another picture for this movie. Just save in assets [this picture from imdb](https://www.imdb.com/title/tt2080374/mediaviewer/rm2977623552?ref_=tt_ov_i) and set link on it instead of using existing one.